src/library/EndpointManagement/EndpointManagementClient.cs

```diff
@@ -494,6 +454,84 @@ private async Task EnsureEndpointManagerRunningAsync(CancellationToken cancellat
             }
         }
 
+        public Dictionary<string, string> GetEnvironmentVariables()
+        {
+            if (!string.IsNullOrEmpty(_environmentVariables.DotNetRoot))
+            {
+                _log.Info("Setting the '{0}' environment variable with '{1}' while launching '{2}'.", EnvironmentVariables.Names.DotNetRoot, _environmentVariables.DotNetRoot, EndpointManager.ProcessName);
+                return new Dictionary<string, string>() { { EnvironmentVariables.Names.DotNetRoot, _environmentVariables.DotNetRoot } };
+            }
+
+            return null;
+        }
+
+        public async Task<(string fileName, string command)> GetEndpointManagerLaunchArguments(string currentUserName, string logFileDirectory, CancellationToken cancellationToken)

```

Comment:
Should this be `private` instead of `public`?

Reply:
done.